src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -615,6 +615,9 @@ CharMap =
 #    mpegps   MPEG Program Stream                           video/mpeg                         Used in DVDs and VCDs
 #    mpegts   MPEG Transport Stream                         video/mpeg                         Used in sat TV and Blu-ray Discs
 #    mpegts   MPEG Transport Stream                         video/vnd.dlna.mpeg-tts            Alternative MIME-type for MPEG TS
+#    oga      Ogg container with only audio content         audio/ogg
+#    ogg      Ogg container                                 video/ogg                          Applies for Vorbis I files only, use with FLAC and Theora should use oga and ogv respectively
+#    ogv      Ogg container with video content              video/ogg
```

Comment:
@Sami32 You'll have to tell me what is waiting for review, because I went through and reviewed those I thought were ready and that I'm capable of reviewing yesterday.